app/assets/javascripts/modules/track-submit.js

```diff
@@ -0,0 +1,21 @@
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function(Modules) {
+  "use strict";
+
+  Modules.TrackSubmit = function () {
+    this.start = function (element) {
+      element.on('submit', 'form', trackSubmit);
+
+      var options = {nonInteraction: 1}, // automatic events shouldn't affect bounce rate
+          category = element.data('track-category'),
+          action = element.data('track-action');
```

Comment:
It means a module that looks and behaves like the `auto-track-event` one has an inconsistency. As this is limited to the frontend app I guess it's fine, but if it was moved into static or the toolkit this could catch someone out.